Pollo/Views/PollBuilder/OnboardingView.swift

```diff
@@ -0,0 +1,524 @@
+        secondCorrectView.layer.cornerRadius = correctViewLength / 2
+        secondCorrectView.layer.borderColor = UIColor.white.cgColor
+        secondCorrectView.layer.borderWidth = correctViewBorderWidth
+        secondCorrectView.alpha = 0
+        addSubview(secondCorrectView)
+        
+        secondOptionLabel.text = secondOptionTextRegular
+        secondOptionLabel.font = ._16RegularFont
+        secondOptionLabel.textColor = .white
+        secondOptionLabel.alpha = 0
+        addSubview(secondOptionLabel)
+        
+        addOptionView.backgroundColor = .clear
+        addOptionView.clipsToBounds = true
+        addOptionView.layer.cornerRadius = optionViewCornerRadius
+        addOptionView.layer.borderColor = UIColor.white.cgColor
+        addOptionView.layer.borderWidth = optionViewBorderWidth
+        addOptionView.alpha = 0
+        addSubview(addOptionView)
+        
+        plusLabel.text = plusLabelText
+        plusLabel.textColor = .white
+        plusLabel.textAlignment = .center
+        plusLabel.font = ._20BoldFont
+        plusLabel.alpha = 0
+        addSubview(plusLabel)
+        
+        addOptionLabel.text = addOptionLabelText
+        addOptionLabel.textColor = .white
+        addOptionLabel.font = ._16RegularFont
+        addOptionLabel.alpha = 0
+        addSubview(addOptionLabel)
+        
+        animatedLabel.textColor = .white
+        animatedLabel.font = ._20BoldFont
+        animatedLabel.numberOfLines = 0
+        animatedLabel.alpha = 0
+        addSubview(animatedLabel)
+        
+        buttonView.backgroundColor = .clear
+        buttonView.clipsToBounds = true
+        buttonView.layer.cornerRadius = buttonViewHeight / 2
+        buttonView.layer.borderWidth = buttonViewBorderWidth
+        buttonView.layer.borderColor = UIColor.white.cgColor
+        buttonView.alpha = 0
+        addSubview(buttonView)
+        
+        buttonLabel.text = startQuestionText
+        buttonLabel.textColor = .white
+        buttonLabel.font = ._16SemiboldFont
+        buttonLabel.textAlignment = .center
+        buttonLabel.alpha = 0
+        addSubview(buttonLabel)
+
+        transitionButton.backgroundColor = .clear
+        transitionButton.addTarget(self, action: #selector(transitionButtonTapped), for: .touchUpInside)
+        addSubview(transitionButton)
+    }
+    
+    func getAnimatedLabelText() -> String {
+        if stage == .createQuestion {
+            return "To customize this question, type here and add additional options"
+        } else if stage == .autofillChoices {
+            return "To autofill choices A, B, ... , tap here to start live polling"
+        } else if stage == .quizMode {
+            return "To use quiz mode, select a correct answer"
+        } else if stage == .saveDraft {
+            return "To come back to this question, tap here"
+        } else if stage == .startQuestion {
+            return "To start polling now, tap here"
```

Comment:
With so many if else's here, should we use a switch instead and for `case .welcome` have it be the empty string?

Reply:
+1